build.psm1

```diff
@@ -579,22 +579,25 @@ function Start-BuildPowerShellNativePackage
         <version>{0}</version>
         <authors>Microsoft</authors>
         <owners>Microsoft,PowerShell</owners>
-        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Native binaries for PowerShell Core</description>
-            <projectUrl>https://github.com/PowerShell/PowerShell</projectUrl>
-            <iconUrl>https://github.com/PowerShell/PowerShell/blob/master/assets/Powershell_black_64.png?raw=true</iconUrl>
-            <license type="expression">MIT</license>
-            <tags>PowerShell</tags>
-            <language>en-US</language>
-            <copyright>© Microsoft Corporation.</copyright>
```

Comment:
is the all rights reserved  required because we aren't supposed to use that.

Reply:
Correct. same as PS/PS.